external/vcm/tests/test_cubedsphere.py

```diff
@@ -260,27 +260,16 @@ def input_dataset(input_dataarray):
 
 @pytest.mark.parametrize("reduction_function", [np.mean, np.median])
 @pytest.mark.parametrize("use_dask", [False, True])
-def test_xarray_block_reduce_dataarray(reduction_function, use_dask, input_dataarray):
-    block_size = (2, 2, 1)
-    expected_data = skimage_block_reduce(
-        input_dataarray.values, block_size=block_size, func=reduction_function
-    )
-    expected = xr.DataArray(
-        expected_data,
-        dims=input_dataarray.dims,
-        coords=None,
-        name="foo",
-        attrs={"units": "m"},
-    )
-
+def test_xarray_block_reduce_dataarray(
+    reduction_function, use_dask, input_dataarray, regtest
+):
     if use_dask:
         input_dataarray = input_dataarray.chunk({"x": 2, "y": 2, "z": -1})
-
     block_sizes = {"x": 2, "y": 2}
     result = _xarray_block_reduce_dataarray(
         input_dataarray, block_sizes, reduction_function
     )
-    assert_identical_including_dtype(result, expected)
+    print(vcm.testing.checksum_dataarray(result), file=regtest)
```

Comment:
It looks like `checksum_dataarray` only takes the hash of the values of the DataArray (and ignores coordinate values and attributes).  Propagating attributes in particular was an issue with the coarsening functions in the past (#513), so I think we may want to revert this change (or modify the way `checksum_dataarray` operates).

Reply:
yah i can do that.